suite-native/app/src/hooks/useFormattersConfig.ts

```diff
@@ -4,16 +4,19 @@ import { useMemo } from 'react';
 import { FormatterProviderConfig } from '@suite-common/formatters';
 import { PROTO } from '@trezor/connect';
 import { selectFiatCurrency } from '@suite-native/module-settings';
+import { selectCoins } from '@suite-common/wallet-core';
 
 export const useFormattersConfig = (): FormatterProviderConfig => {
     const fiatCurrency = useSelector(selectFiatCurrency);
+    const coins = useSelector(selectCoins);
```

Comment:
Same problem as mentioned in desktop version of this hook.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/a684fb2a201e098bf6f44c05617359fb798ba2fe